public/translations/EN.json

```diff
@@ -0,0 +1,50 @@
+{
+    "home.head.title": "Sprikipedia, the Sprig encyclopedia",
+    "home.welcome.welcome": "Welcome to Sprikipedia,",
+    "home.welcome.description": "the Sprig encyclopedia that anyone can edit.",
+    "home.docs.start.name": "Start with Sprig!",
+    "home.docs.start.description": "Starting with Sprig? Here is the perfect place",
+    "home.docs.tutorials.name": "Tutorials",
+    "home.docs.tutorials.description": "Step-by-step to make things for your game",
+    "home.docs.firmware.name": "Firmware",
+    "home.docs.firmware.description": "Links to Spade, Trowel, Spryg...",
+    "home.docs.hardware.name": "Hardware",
+    "home.docs.hardware.description": "Link to our PCB, components...",
+    "home.docs.team.name": "Team",
+    "home.docs.team.description": "Who make Sprig possible",
+    "home.docs.about.name": "About Sprikipedia",
+    "home.docs.about.description": "The history, purpose, everthing",
+    "home.docs.brand.name": "Brand",
+    "home.docs.brand.description": "All branding material for download",
+    "home.dark.info": "Come to the dark side!",
+    "about.head.title": "About Sprikipedia - Sprikipedia",
+    "about.content.title": "About Sprikipedia",
+    "about.blockquote.quote": "Imagine a world in which every single person on the planet is given free access to the sum of all Sprig knowledge. That's what we're doing.",
+    "about.blockquote.citation": "Lucas, rephrasing Jimmy Wales quote from Wikipedia's about page",
+    "about.content.p1": "Sprikipedia is a encyclopedia.",
+    "about.content.p2": "Sprikipedia's purpose is to benefit readers by containing information on all branches of Sprig, Written collaboratively by members of the <a href="https://hackclub.com">Hack Club Community</a>. Since its creation, as a <a href="https://github.com/hackclub/sprig/wiki">wiki page</a> in <a href="https://github.com/hackclub/sprig/">our GitHub repository</a>, on March 13, 2023 by Graham, it has grown to cover all things Sprig.",
```

Comment:
It seems like you would be mixing the presentation/UI layer with the localization layer here, which is probably not desirable.  Suggest keeping these cleanly separate.

Reply:
done!